motivation.Rmd

```diff
@@ -0,0 +1,111 @@
+# Motivation
+
+## Language barrier
+
+> The predominant language of open source is English—in
+code, content, and community interactions—and English
+proficiency is a metric by which performance and
+personality can be judged. [^1] 
+
+The Linux Foundation report also identify _language 
+access_ as one of the _environmental Barriers to Equity 
+in Open Source_.
+
+Publishing multilingual resources can lower these
+barriers by _increasing access to knowledge_, which helps
+_democratize access to quality resources_, and therefore
+_increases the potential for contributing_ to software 
+and open science projects.
+
+For this reason, we are embarking on an effort to 
+translate rOpenSci’s material on best
+practices for software development, code review, and 
+contribution to open source projects beyond the English
+language starting with _Spanish_.
+
+Spanish is the second most-spoken native language in the 
+world. It is also one of the most geographically
+widespread languages, reaching many countries with 
```

Comment:
source? especially if the source has a map :smiley:

Reply:
Si, es este artículo! lo agrego. https://en.wikipedia.org/wiki/List_of_languages_by_number_of_native_speakers